### ISSUE TITLE
Large package size

### ISSUE THREAD

**Issue by Thane5** [peeweek/net.peeweek.gameplay-ingredients#80]
I noticed that this Package creates a folder in my package cache that is 61.4mb large. 59.4 of that are just the documentation. 
Is there a way to stop Unity from downloading that folder?

**Comment by peeweek** [comment]
Indeed, I have not cleaned the documentation folder as it contains many source and project files. I will try to find a solution in OpenUPM to move them to an excluded folder,
